### ISSUE TITLE
Questions about Mac and Linux dlls

### ISSUE THREAD

**Issue by Yousei9** [tux3/LoE-PrivateServer#93]
Could someone please tell me if and where there are differences in the patched dlls for Win and Mac/Linux.

Background: 
I tried to patch the game to connect direct to my game server without running a local p-server. It works fine on win, but mac and linux clients get stuck after login.

**Comment by DanielOaks** [comment]
Nah, no difference at all. If anything, it'd probably just be a version difference.

Just as a note, this doesn't work with the latest clients, since the LoE devs basically rewrote most of the networking for it.

**Comment by Yousei9** [comment]
Thanks.
Sadly that means I'm now out of ideas why mac babscon 2014 clients don't work while win babscons work flawlessly with my dlls.

**Comment by EpicLPer** [comment]
Well, easy: Stop using Apple ;P

(jk)

**Comment by Yousei9** [comment]
Yeah that's my usual comment to anyone using that crap.

**Comment by PyroSamurai** [comment]
I may be a matter of the port being blocked by the mac firewall. Since I assume you used a different computer for the mac and windows clients. On GNU+Linux low numbered ports would be block (1024 and below) and mac is a similar OS so it probably does something similar. The bug you are talking about was suppose to be taken care of in the babscon release by moving the connection port to 1034, so I'd check any firewalls you may have in the way of the connection.
https://github.com/tux3/LoE-PrivateServer/releases/tag/0.5.3

**Comment by DanielOaks** [comment]
Yeah, that could cause something actually. I made the server work for OSX,
but I've never tried with using the OSX client on a Windows server or
something similar (the client did work flawlessly with the unofficial
servers while they were running, though).

I'll give it a shot sometime in the next day or two, make sure Babscon 2014
works with it all.

**Comment by Yousei9** [comment]
Sorry I didn't reply sooner, i wanted to test it myself and couldn't get osx to run in a vm until now.
The dll worked flawlessly for myself in the vm osx. 
Some players had the bug where it lists down to Atlas.. Done and then get's stuck though. Where does OSX store CutieMarks.xml, Atlas.png, asf?
